constrained-voucher.mkd

```diff
@@ -265,8 +265,14 @@ The MASA needs to support all formats that the Pledge, produced by that manufact
 
 ## Extensions to BRSKI {#brski-extensions}
 
-A Pledge that only supports the EST-coaps enrollment method SHOULD NOT use discovery for BRSKI resources, since it is more efficient to just try
-the supported enrollment method via the well-known BRSKI/EST-coaps resources, and it avoids the Pledge having to do complex CoRE Link Format parsing.
+The Pledge discovers an IP address and port number that connects to the Registrar (possibly via a Join Proxy), and it establishes a DTLS connection.
+
+No further discovery of hosts or port numbers is required, but a pledge that can do more than one kind of enrollment (future work offers protocols other than {{EST-coaps}}), then a pledge may need to use CoAP Discovery to determine what other protocols are available.
+
+However, a Pledge that only supports the EST-coaps enrollment method SHOULD NOT use discovery for BRSKI resources, since it is more efficient to just try the supported enrollment method via the well-known BRSKI/EST-coaps resources, and it avoids the Pledge having to do complex CoRE Link Format parsing.
+
+In order to support this, the Registrar MUST support all of the EST resources at their default "well-known" locations (on the port number discovered) as well as any shorter form that the might also be supported.
```

Comment:
The part about "as well as any shorter form .... " here is unclear to me. Whether any shorter-form resource is supported or not by the server on the currently connected (DTLS) port number is something that the server may choose. If it supports a short-form resource, it would typically be discovered by the client.
* It could have been discovered prior to DTLS-connecting. (Peter's model - where unsecured CoAP discovery prior to bootstrapping is allowed.)
* It could have been discovered after doing the DTLS connection - in this case a GET request is sent to /.well-known/core using the same DTLS connection i.e. request to the same port as already DTLS-connected to, typically.  (Although a separate request to port 5683 is also possible in some deployments, e.g. Peter's)